automation_testing/utils/urls/environments.ts

```diff
@@ -0,0 +1,10 @@
+const env = {
+    local: 'http://kube.simpleview.io:10000/iframe.html',
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
	local: 'http://kube.simpleview.io:10000/iframe.html',
```